src/dev/lcd_hd44780.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#ifndef DSY_LCD_HD44780_H
+#define DSY_LCD_HD44780_H
+
+#include "daisy_core.h"
+#include "per/gpio.h"
+
+
```

Comment:
Add a brief description in doxygen format to describe the class, and provide any specific hardware details that may not be immediately obvious to a user.

+1 for a url to an LCD display someone can order that'll work with this code.

Reply:
https://wiki.dfrobot.com/Arduino_LCD_KeyPad_Shield__SKU__DFR0009_

Reply:
I am not familiar with the doxygen format, could you provide me with a link? Sorry.

Reply:
Do you want the functions to enable/disable blinking and the cursor right now, or could it wait for 2.0? :)

Reply:
@StaffanMelin Talking in multiple places at once can be a bit confusing. My bad.  In our direct messages I sent you a link to another libdaisy file  you can use as a reference for the doxygen stuff.

And yeah, I'm totally fine with skipping the enable/disable functions for now :)